summit/2018.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+permalink: /summit/
```

Comment:
I wonder if this should be /summer/2018 to make it more permanent?

Reply:
I think it makes a bit more sense to keep the current summit and archive it later on. It also has a nicer URL. I can see though that theoretically things could break in the future. I don't know, have no strong opinion on this.

Reply:
I believe we should keep it summit for each present edition and then archive it as /summit/{{year}}. Each summit page can have a link to the past or newer summit.

Reply:
I guess @davelab6 meant /summit/ instead of /summer/ ;)

For the potential attendees coming looking specifically for the summit, you could argue /YYYY makes sense.

However, since we organise more than just the summit (FOSDEM devroom, design clinic), I'm leaning towards @davelab6's suggestion. It is also consistent with our other URL structure:

osd.net/events

So, with this we'd have:

osd.net/events
osd.net/summit 

---------

Looking at how other conferences/summits do it:

Fosdem (you can argue it *ONE* event) has an archive for previous years:

archive.fosdem.org/2017
archive.fosdem.org/2016
...
Froscon have similar structure:

https://programm.froscon.de/2017/
https://programm.froscon.de/2016/
-------

Also is there a reason why we can't "swap" the ordering of the support-us and location sections?

<section id="support-us" class="content-section text-center"> 
</section> 

<section id="location" class="content-section text-center"> 
</section> 

so that it becomes:

<section id="location" class="content-section text-center"> 
</section> 

<section id="support-us" class="content-section text-center"> 
</section> 

again, from the point of view of the user who is looking for information, finding out where the summit is is more important than supporting us (which is obviously important to us!).

Reply:
I have already changed the order of the sections in my latest commits :)

Reply:
Yep just saw that now. 👍

Reply:
One final question: 

what location/building/address is the map of? If it is a real map, then we should have this in the location too?

Or is the map just a holding image?

Reply:
OK gotcha, so whats the real address of the venue?

Reply:
The venue is not set yet. It's a potential candidate out of 2 but we don't think it should be set as of now without discussing it first

Reply:
@AnXh3L0 @elioqoshi Yep, OK.

The only other update I'd suggest then is making it clear that the actual venue isn't set.

so something like:

in <section id="location" class="content-section text-center">

<p class="data">Venue still to be confirmed, Tirana, Albania</p>

...apart from that I think the content and its organisation is good. 👍